cli/types/index.d.ts

```diff
@@ -1803,6 +1803,19 @@ declare namespace Cypress {
     whitelist: string | string[] | RegExp | ((cookie: any) => boolean)
   }
 
+  /**
+   * Options that control how a command behaves in the `within` scope
+   */
+
+  interface Withinable {
+    /**
+     * The subject of the within wrapper.

```

Comment:
This could probably be documented better. Is the following accurate?

> Element to search for children in. If `null`, search begins from root-level DOM element.

Reply:
Yeah that looks better for sure.